superset-frontend/src/middleware/loggerMiddleware.js

```diff
@@ -81,7 +81,9 @@ const loggerMiddleware = store => next => action => {
   const { eventName } = action.payload;
   let { eventData = {} } = action.payload;
 
-  if (dashboardInfo?.id && eventData.path?.includes('/dashboard/')) {
+  const path = location?.href;
```

Comment:
Maybe preserve the previous logic in case the event includes the path?
```suggestion
  const path = eventData.path || location?.href;
```